_workinggroups/quantumcomputing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Quantum Computing"
+layout: plain
+---
+
+# The HSF Quantum Computing working group
+
+## Introduction
+
+The Quantum Computing Working Group fosters collaboration among the HEP experiments and institutes to exploit the possibilities offered by quantum computing.
+
+Quantum computing is a very active research field and prototype computers, simulators, and toolkits are now available. The participants in the working group share experiences, ideas, know-how, demonstrators, code snippets, and documents with the aim of fostering common research and development in this field, for the benefit of the HEP community.
+
+
+[Mandate](/organization/working-group-mandates.html).
+
+Kick-starters: Riccardo Maria BIANCHI (ATLAS), Sebastien BINET (LSST, ALICE, LPC), Riccardo DI SIPIO (ATLAS), Lukas HEINRICH (ATLAS), Maurizio PIERINI (CMS)
+
+----
+
+## Repository
+
+You can find code snippets and demonstrators in the working group's GitHub repository: <https://github.com/HSF/quantum-computing>. All code, unless differently specified, is released under the Apache 2.0 license. 
```

Comment:
I think the Apache license is not correct here - that should be in the code repository itself.

Reply:
Hi Graeme,
Thanks for your comments and suggestions.

I have the LICENSE file in the QC repo, already: https://github.com/HSF/quantum-computing 

Sorry, but I didn't really understand your comment: should I remove the mention to the license here and leave the license in the repository only?

Best,
--    Ric.

Reply:
Yes, it's not relevant to mention it here. Keep the it in the repo where it belongs.

Reply:
OK, thanks.
Done.